packages/lesswrong/components/sunshineDashboard/SunshineCuratedSuggestionsList.tsx

```diff
@@ -29,8 +33,13 @@ const SunshineCuratedSuggestionsList = ({ terms, belowFold, classes }:{
   const curatedDate = curatedResults ? new Date(curatedResults[0]?.curatedDate) : new Date();
   const twoAndAHalfDaysAgo = new Date(new Date().getTime()-(2.5*24*60*60*1000));
 
-  if (!belowFold && (curatedDate > twoAndAHalfDaysAgo)) return null
-  
+  if (
+    (!belowFold && (curatedDate > twoAndAHalfDaysAgo)) ||
+    (forumTypeSetting.get() === "EAForum" && !currentUser?.isAdmin)
```

Comment:
Do you display the curated suggestions  component for non-admins?

Reply:
ah, I guess you have people with `posts.moderate.all` that you don't want to be able to curate things?

Reply:
Yep - that's exactly the issue!

Reply:
I have a sense that this is an example of different-forum-different-function code which would be nice to refactor slightly in some way, although I'm not 100% sure which way.

I think it'd be at least a slight improvement if you broke each condition out into a variable with names like "defaultHidingBehavior" and "eaForumHidingBehavior" or something.

I feel like there's a possible improvement beyond that but can't think of it offhand.